dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java

```diff
@@ -286,6 +287,68 @@ public Map<Partner, Metrics> aggregatedMetricsGet(@ApiParam(hidden = true) @Para
         return delegate.getAggregatedMetrics(id, versionId, user);
     }
 
+    @GET
+    @UnitOfWork(readOnly = true)
+    @RolesAllowed({"curator", "admin", "platformPartner"})
```

Comment:
If a user is a `platformPartner`, couldn't they update/see all platform partners' raw data, instead of just their own platform? I know there's nothing sensitive in the raw data we've seen so far, but that seems potentially risky. Even though we only have one partner interested so far in invoking the API.

If I'm right, I suggest we remove the `platformPartner` role for the 1.15 release, then adding it back in for 1.15.x or 1.16, once we have partner that wants to start hitting our API. Or we make sure we only allow one partner max.

Reply:
Yes, you're right.

Another option is to track the platform for a platformParter user in the `User` class, which would only involve adding a column and doing a check in the applicable endpoints.

Reply:
Updated the PR so that a platform partner user can only access/modify/submit executions for their own platform

Reply:
@coverbeck I'm merging this PR, but please let me know if you have any strong objections to the changes since your last approval

Reply:
> @coverbeck I'm merging this PR, but please let me know if you have any strong objections to the changes since your last approval

@kathy-t Changes look good, thanks!